client/payment-details/summary/index.tsx

```diff
@@ -177,6 +186,16 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 
 	const isFraudOutcomeReview = isOnHoldByFraudTools( charge, paymentIntent );
 
+	const disputeFee =
+		charge.dispute &&
+		charge.dispute.status !== 'won' &&
```

Comment:
Added this criteria for rendering the tooltip breakdown to prevent the fee rendering for `won` disputes.

<img width="1074" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/3de07f0b-1f5b-486f-a1a8-ba0a8456d531">